src/mcp/client/session.py

```diff
@@ -140,6 +140,7 @@ async def initialize(self) -> types.InitializeResult:
                 )
             ),
             types.InitializeResult,
+            cancellable=False,
```

Comment:
slightly more wordy name for this variable might be "propagate_cancel_to_server" perhaps call this out in docstring?